internal/flags/helpers.go

```diff
@@ -17,11 +17,10 @@
 package flags
 
 import (
+	"github.com/ethereum/go-ethereum/params"
```

Comment:
the `github.com/...` imports should be in a separate second group, and the 'native' ones should be in the first group

Reply:
Fixed.